extensions/wikia/AdEngine/js/video/player/porvata/googleImaPlayerFactory.js

```diff
@@ -33,6 +33,14 @@ define('ext.wikia.adEngine.video.player.porvata.googleImaPlayerFactory', [
 			}
 		}
 
+		function setAutoplay(value) {
+			// videoAd DOM element is present on mobile only
+			if (videoAd) {
+				videoAd.autoplay = value;
+				videoAd.muted = value;
```

Comment:
It's a requirement for autoplay on mobile:
https://developers.google.com/web/updates/2016/07/autoplay